style.css

```diff
@@ -0,0 +1,47 @@
+h1 {
+  font-size: 200px;
+  color: white;
+ 
+  
+}
+
+h2 {
+  color: white;
+  text-align: left;
+ 
+   
+
+}
+body {
+  
+  font-family: "Helvetica Neue", sans-serif;
+  font-size: 24px;    
+  
+  
+}
+
+P {
```

Comment:
All CSS should be written [lowercase](https://mica-web.github.io/learn/css/best-practices#write-your-css-all-lowercase).